### ISSUE TITLE
541

### ISSUE THREAD

**Issue by PaymonK** [Geeksltd/GCop#81]
Currently the entire method gets error wiggles underlines. Only the `async` keyword should.

**Comment by PaymonK** [comment]
Also this is a false positive:

```
 [HttpPost, Authorize, Route("upload")]
        public async Task<IActionResult> UploadTempFileToServer(IFormFile[] files)
        {
            return Json(await new FileUploadService().TempSaveUploadedFile(files[0]));
        }
```

The await here is not right after `return` and therefore the rule should not apply.

**Comment by Foroughi** [comment]
@reporter : Fixed on[ NuGetPackages\25ce933ff240d73db0e7a32789b608ae48c7aed6](https://gitlab.com/Paymon/GCop/tree/25ce933ff240d73db0e7a32789b608ae48c7aed6)

@Karvan  That is already fixed on #78
